osbenchmark/builder/installers/exception_handling_installer.py

```diff
@@ -0,0 +1,20 @@
+from osbenchmark.builder.installers.installer import Installer
+from osbenchmark.exceptions import InstallError
+
+
+class ExceptionHandlingInstaller(Installer):
+    def __init__(self, installer, executor=None):
+        super().__init__(executor)
+        self.installer = installer
+
+    def install(self, host, binaries, all_node_ips):
+        try:
+            return self.installer.install(host, binaries, all_node_ips)
+        except Exception as e:
+            raise InstallError("Installing node on host \"{}\" failed".format(host), e)
```

Comment:
Nit: I think f-strings are the standard now for Python string formatting. Sorry if I missed this on other PRs

`raise InstallError(f"Installing node on host \"{host}\" failed", e)`

Reply:
Didn't know that. Will fix the instances here then swap out the rest of the occurrences in a followup PR